SDWebImage/SDImageCache.h

```diff
@@ -323,12 +323,12 @@ typedef NS_OPTIONS(NSUInteger, SDImageCacheOptions) {
 /**
  * Get the size used by the disk cache
  */
-- (NSUInteger)getSize;
+- (NSUInteger)totalDiskSize;
```

Comment:
I agree to rename some properties. I suggest some name to discussion.
See `NSURLCache`, it uses `diskCapacity`, the term `size` may more suitable to express two or three dimension.

`totalDiskCount`, TBO, if I don't see comment, I can't ensure what it means, because I don't know `diskCount`. The property means total images count on disk, `countOfImagesOnDisk`, just for discussion.

Reply:
The one naming I though, it's to use `totalDiskCacheSize`. But I think it's a little dummy because we're already using the SDImage**Cache** and everyone know it have a disk cache.

YY also name it simple, it's enough to distinguish from the memory cache representation.

https://github.com/ibireme/YYCache/blob/master/YYCache/YYDiskCache.h#L96

Reply:
Size can mean two dimension in geometry. But however, in the concept about memory. We always use `heap size`, `memory size`, `bytes size` to point to the bytes we used.

Actually these two method, just renaming it to not to use `get` word. It's not suitable because of Objective-C API naming guideline. So I remove the `get` with `totalDisk`, other word is kept.

Reply:
@dreampiggy Seems you really love `YY*` 😂 , it's good but not the best, right? Apple already told us how to name, it provides `NSURLCache`, you can check it and also I suggest you to investigate other popular frameworks.